### PR TITLE
Review of CITATION.cff for the LTS release

### PR DESCRIPTION
**Pull request by bkhelifi**
**Description**
In the context of preparation of our first LTS release, **the author list should be reviewed by each of the Gammapy contributors**. Emails will also be sent as reminder.

**Dear reviewer**
Followingthe PIG 24 #3970, the authorship policy for the v1.0 release is special, as the DCO mechanism (see #4035) was not installed since the beginning of the project.

So, each contributor should comment this PR by:
1. if she/her wishes to "OptOut", removing her/his associated data
2. checking her/his name (first and family) and the information about email address
4. checking/adding/removing her/his ORCID (we do recommend to have one and to update your personal information, as Gammapy follows the EU Open Science recommendations)
5. checking/adding/removing her/his affiliation and its associated city, accordingly to your employer agreement and rules _at the time_ of your contribution or latest contribution.

In case of no reaction or answer, the current information will be kept.

### CODE REVIEW

File: CITATION.cff [hunk 1]
```diff
@@ -1,290 +1,431 @@
...
+  affiliation: Technical University of Dortmund
+  city: Dortmund, DE
+- family-names: Buchner
+  given-names: Johannes
+  email: johannes.buchner.acad@gmx.com
+  orcid: https://orcid.org/0000-0003-0426-6634
+  affiliation: Max Planck Institute for extraterrestrial Physics
+  city: Garching, DE
+- family-names: Cabot
+  given-names: Arnau Aguasca
```
Comment (aaguasca) [inline CITATION.cff:81]: It should be the following:
family-names: Aguasca-Cabot
given-names: Arnau
Thanks!

**Review by aaguasca — COMMENTED** [review]
There is a small error in my family and given name.

File: CITATION.cff (hunk 1 above)
Comment (bkhelifi) [reply CITATION.cff:81]: Thanks! Is your affiliation correct?

Comment (aaguasca) [reply CITATION.cff]: I would prefer to have this one: "Departament de Física Quàntica i Astrofísica, Institut de Ciències del Cosmos, Universitat de Barcelona, IEEC-UB".
Thank you very much.

File: CITATION.cff [hunk 2]
```diff
@@ -1,290 +1,431 @@
...
 authors:
-- given-names: Axel
-  family-names: Donath
-  affiliation: Center for Astrophysics | Harvard & Smithsonian, CfA, 60 Garden St.,
-    02138 Cambridge MA, US
+- family-names: Acero
+  given-names: Fabio
+  email: fabio.acero@cea.fr
+  orcid: https://orcid.org/0000-0002-6606-2816
+  affiliation: AIM (DAp)
```
Comment (facero) [inline CITATION.cff:42]: Hi @bkhelifi ,
Could you update my affiliation to the following one:
Université Paris-Saclay, Université Paris Cité, CEA, CNRS, AIM de Paris-Saclay, 91191 Gif sur Yvette

and the city to Gif sur Yvette
ORCID number is correct.
thanks !

File: CITATION.cff [hunk 3]
```diff
@@ -1,290 +1,431 @@
...
+  city: Heidelberg, DE
+- family-names: Ruiz
+  given-names: José Enrique
+  email: jer@iaa.es
+  orcid: https://orcid.org/0000-0003-3274-4445
+  affiliation: Instituto de Astrofísica de Andalucía - CSIC
+  city: Granada, Spain
+- family-names: Saha
+  given-names: Lab
+  email: labsaha@ucm.es
```
Comment (AtreyeeS) [inline CITATION.cff:332]: @labsaha  is not at the UCM anymore.

File: CITATION.cff [hunk 4]
```diff
@@ -1,290 +1,431 @@
...
+- family-names: Siejkowski
+  given-names: Hubert
+  email: hubert.siejkowski@gmail.com
+  affiliation: CYFRONET AGH
+  city: Krakow, PL
+  orcid: https://orcid.org/0000-0003-1673-2145
+- family-names: Sinha
+  given-names: Atreyee
+  email: asinha@ucm.es
+  affiliation: UCM, IPARCOS
```
Comment (AtreyeeS) [inline CITATION.cff:345]:
```suggestion
  affiliation: IPARCOS, UCM
```

File: CITATION.cff (hunk 3 above)
Comment (bkhelifi) [reply CITATION.cff:332]: At the time of the contribution, it was... I will contact Lab to ask

File: CITATION.cff [hunk 5]
```diff
@@ -35,397 +35,173 @@ contact:
...
+    orcid: "https://orcid.org/0000-0002-3882-9477"
+  - given-names: Lars
+    family-names: Mohrmann
+    affiliation: "Max Planck Institute for Nuclear Physics, Heidelberg"
+    orcid: "https://orcid.org/0000-0002-9667-8654"
+  - given-names: Daniel
+    family-names: Morcuende
+    affiliation: "EMFTEL department and IPARCOS, Universidad Complutense de Madrid, 28040 Madrid, Spain"
+    orcid: "https://orcid.org/0000-0001-9400-0922"
+  - given-names: Daniel
```
Comment (adonath) [inline CITATION.cff:119]: This entry is duplicated...

**Review by adonath — APPROVED** [review]
Thanks @bkhelifi!

File: CITATION.cff [hunk 6]
```diff
@@ -1,290 +1,207 @@
...
+    family-names: Ruiz
+    affiliation: "Instituto de Astrofísica de Andalucía – CSIC, Spain"
+    orcid: "https://orcid.org/0000-0003-3274-4445"
+  - given-names: Hubert
+    family-names: Siejkowski
+    affiliation: "ACC Cyfronet AGH, Poland"
+    orcid: "https://orcid.org/0000-0003-1673-2145"
+  - given-names: Atreyee
+    family-names: Sinha
+    affiliation: "Departamento de EMFTEL, IPARCOS, Universidad Complutense de Madrid, 28040 Madrid, Spain"
```
Comment (AtreyeeS) [inline CITATION.cff:172]:
```suggestion
    affiliation: "EMFTEL department and IPARCOS, Universidad Complutense de Madrid, 28040 Madrid, Spain"
```

File: CITATION.cff [hunk 7]
```diff
@@ -1,290 +1,207 @@
...
+    family-names: Linhoff
+    affiliation: "Astroparticle Physics Group, TU Dortmund University, Germany"
+    orcid: "https://orcid.org/0000-0001-7993-8189"
+  - given-names: Ruben
+    family-names: Lopez-Coto
+    affiliation: "IAA-CSIC, Spain"
+    orcid: "https://orcid.org/0000-0002-3882-9477"
+  - given-names: Lars
+    family-names: Mohrmann
+    affiliation: "Max Planck Institute for Nuclear Physics, Heidelberg"
```
Comment (AtreyeeS) [inline CITATION.cff:113]:
```suggestion
    affiliation: "Max Planck Institute for Nuclear Physics, Heidelberg, Germany"
```

File: CITATION.cff [hunk 8]
```diff
@@ -1,290 +1,207 @@
...
+    family-names: Vuillaume
+    affiliation: "Univ. Savoie Mont-Blanc, LAPP, CNRS, Annecy, France"
+    orcid: "https://orcid.org/0000-0002-5686-2078"
+  - given-names: Amanda
+    family-names: Weinstein
+    affiliation: "Iowa State University, USA"
+    orcid: "https://orcid.org/0000-0002-2126-2419"
+  - given-names: Matthew
+    family-names: Wood
+    affiliation: "SLAC, USA"
```
Comment (AtreyeeS) [inline CITATION.cff:207]: SLAC is the only institute with abbreviation in this list...

**Review by registerrier — COMMENTED** [review]
Thanks @bkhelifi . 
The date in the codemeta.json should be changed to November 9th.
Why is Fabio listed as the maintainer?

**Pull request merged**